Magento2/Tests/Less/ColourDefinitionUnitTest.less

```diff
@@ -0,0 +1,24 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+@red: #aaaaaa;
+@blue: #00F;
+@green: #0f0;
```

Comment:
Can you please add a correct variable definition, i.e. `@purple: #900c3f`

Reply:
`@green` is a correct variable definition

Reply:
Done @sivaschenko